MetaMask/metamask-mobile
# [FIX] Transaction of Matic submitted in polygon network goes to the activity view

**Description**
The problem was that the submitted transactions in the Polygon Mainnet do not appear in the left option Activity View.
The network firstly was always loading, then the first submitted transaction was not appearing on the activity view.
The transactions were not appearing because we do not render the submitted transactions when we do not have any other listed transaction

**Proposed Solution**
The proposed solution it's showing as well the transactions submitted when we have no older transactions

**Code Impact**
Low, it will only impact the Activity View screen.

**Test Cases**
Case:1
- Go to settings -> Advanced
- Press reset account (to have no transaction on activity view)
- Go to polygon network
- transfer 0.001 matic
- go to activity view and verify if it's there
(Did the same for rinkeby network)
Case2:
- Tested transfer on rinkeby network
Case3:
- Tranfer of collectible on rinkeby network


**Checklist**


**Screenshots/Recordings**

https://user-images.githubusercontent.com/46944231/168602382-7cde981b-14c3-4d92-a8d4-416d84b18933.mp4


**Issue**

Progresses #3820
